bcda/api/alr.go

```diff
@@ -19,13 +19,13 @@ func (h *Handler) alrRequest(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 
 	// retrieve ACO & cms_id data from the context through value
-	ad, err := readAuthData(r)
+	ad, err := GetCtxAuthData(r)
```

Comment:
Like the new name!  Any thoughts on name: `GetAuthDataFromCtx`?  I feel like its a bit more explicit/clear, but its also a bit longer...  not sure myself which one I like more so thought I would throw it out there.

Reply:
yeah, I came back to that rename with fresh eyes and felt conflicted about the name, since the order is reversed from what it should be. I am always in favor of explicit naming, even if it is longer. I think I'd rather switch this around, so I will make that update!